Explain events go inside target interfaces (#4495)
mdn/browser-compat-data: docs/data-guidelines.md

## DOM events (`eventname_event`)

> Name DOM event features in the form _eventname_\_event with description text `<code>eventname</code> event`. Each event's record is a member of the interface which is its target. If an event can be sent to multiple interfaces, you need to replicate the event in each of those interfaces.

Reviewer: Can we present this an imperative? It's not optional, after all. Like this maybe?

```suggestion
Add DOM events as features of their target interfaces. If an event can be sent to multiple interfaces, then add the event as a feature of each interface. Name DOM event features in the form _eventname_\_event with description text `<code>eventname</code> event`.
```

Author: I've rewritten the paragraph; hopefully it satisfies. It's not the same as what you wrote but is somewhat closer. :)

> For example, the feature for a `focus` event whose target interface is `Element` would be named `focus_event` with the description text `<code>focus</code> event`, like this:

Reviewer: I'm not keen on anthropomorphizing events. How about this? 

```suggestion
For example, the feature for a `focus` event targeting the `Element` interface would be named `focus_event` with the description text `<code>focus</code> event`, like this:
```

Author: I've committed a revision that revises this text similarly to what you suggest.
